conn_windows.go

```diff
@@ -13,7 +13,7 @@ import (
 )
 
 const (
-	disablePathMTUDiscovery = true
+	disablePathMTUDiscovery = false
```

Comment:
We can remove this const (and the associated code path now), right? It's set to `false` on all architectures.

Reply:
undo previous PR that added the checks.